dev-python/reedsolomon/files/cythonize.patch

```diff
@@ -0,0 +1,19 @@
+Patch source https://github.com/tomerfiliba/reedsolomon/pull/38/
+
+Force cythonize of *.pyx file
+
+It seems like cythonize of your *.pyx does not do anything if a *.c file. By using force=True it uses the *.pyx and also fixes the issues on Python 3.10.
```

Comment:
It would be simpler to just `rm` the .c files in ebuild.

Reply:
yes... I am not very familiar with python ebuilds, glad to learn this!